### DISCUSSION TITLE
Fail to install

### DISCUSSION THREAD

**Discussion by ZichengWangdrzz** [ohmyzsh/ohmyzsh#11759] [Help] [answered]
`sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
I'm using WezTerm terminal, keep getting the error information:

`curl: (7) Failed to connect to raw.githubusercontent.com port 443 after 179 ms: Connection refused`

I've tried turn on or off my VPN. Neither of them worked. Someone please help.

**Comment by rwmitchell** [comment]
What country are you in?

**Reply by ZichengWangdrzz** [reply]
I'm in China, right now.

**Reply by rwmitchell** [reply]
Someone else will better comment, but I believe China and India are the two countries blocking githubusercontent.com
If you search thru past issues, i believe there are also suggested solutions.

**Reply by ZichengWangdrzz** [reply]
Will try, thanks a lot.

**Comment by carlosala** [comment, marked as answer]
Use https://install.ohmyz.sh instead👍
